### ISSUE TITLE
Custom lexicon possessive pronouns not registering

### ISSUE THREAD

**Issue by timcosta** [nlp-compromise/compromise#394]
No clue if my title is appropriate for the problem i'm running into, but its the best I can come up with right now. Given the following:
```
NLP('US to withhold funds from Pakistan over failure to fight Taliban's Haqqani Network', {
    Taliban: ['Organization'],
}
```
`Taliban` is identified as a Noun that is Possessive and Plural.
```
{ spaceBefore: ' ',
    text: 'Taliban\'s',
    spaceAfter: '',
    normal: 'taliban\'s',
    implicit: '',
    bestTag: 'Noun',
    tags: [ 'TitleCase', 'Noun', 'Possessive', 'Plural' ] }
```
My custom lexicon doesn't seem to be respected here. I have tried adding a few custom names in other cases, and they do work. It seems to be something about the `'s` that is throwing off the detection in this particular case.

**Comment by spencermountain** [comment]
ah, tim, thank you for the good bug. I totally forgot about this.
I will fix it today

**Comment by spencermountain** [comment]
hey tim, fixed in 10.7.1
https://runkit.com/spencermountain/5974ea97f5b7360011424eec

**Comment by timcosta** [comment]
Thanks so much!!
